cbqueryx/queryrespreader.go

```diff
@@ -174,6 +181,14 @@ func (r *queryRespReader) parseError(errJson *queryErrorJson) *ServerError {
 		err = ErrAuthenticationFailure
 	}
 
+	if errCode == 12021 {
```

Comment:
I modified this file to parse errors in order of error-code so that they are easier to read and easier to find, would you be able to move these up above to be in order?